src/common/catalog-entities/kubernetes-cluster.ts

```diff
@@ -53,8 +53,11 @@ export interface KubernetesClusterStatus extends CatalogEntityStatus {
 }
 
 export class KubernetesCluster extends CatalogEntity<CatalogEntityMetadata, KubernetesClusterStatus, KubernetesClusterSpec> {
-  public readonly apiVersion = "entity.k8slens.dev/v1alpha1";
-  public readonly kind = "KubernetesCluster";
+  static apiVersion = "entity.k8slens.dev/v1alpha1";
+  static kind = "KubernetesCluster";
```

Comment:
This class does not need them, I added them so that other places can use these: https://github.com/lensapp/lens/pull/2909/files#diff-5f80bd77276864216b34ae50b0db19ba251466314ae4cc141f36b8169611c9beR58

Reply:
Couldn't that have been `entity.apiVersion` and `entity.kind`?

Reply:
Oh, of course. Idk what I was thinking here. Changed.